woocommerce/woocommerce-gateway-stripe
# Update link svg image

Fixes #2984

## Changes proposed in this Pull Request:

This PR updates the Link icon and the link URLs in the description

## Testing instructions

1. Go to `WP-Admin > WooCommerce > Payments`
2. Click on the `Stripe` gateway
3. Check that the new icon is visible in the list and the links point to `link.com` instead of the old `link.co`:
![Screenshot 2024-03-15 at 1 32 35 PM](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/407542/786d98f8-7daa-4b49-99db-4ea3402042a8)
4. If `Link` is not present in the list, go to the `Settings` tab and check that `Enable the updated checkout experience` is enabled in the `Advanced settings` section
